<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2463</width>
    <height>931</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="coords">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>541</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Координаты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_OK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Поставить корабли</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>410</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_new_game">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>280</y>
     <width>561</width>
     <height>731</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
